### ISSUE TITLE
Bokeh servers do not start at the right time in Jupyter notebooks

### ISSUE THREAD

**Issue by RiccardoGiro** [holoviz/holoviews#4386]
The code displayed below produces two simple figures to be rendered in two separate windows of the browser. Unfortunately, both figures are shown only after the whole content of the notebook cell has finished running. In theory, the first figure should be displayed immediately, while the second one is generated only after having received a keyboard input from the user. Why does this problem occur?
```
from holoviews import renderer, Curve


bokeh_renderer = renderer('bokeh').instance(mode='server')


# Creation of a figure
figure_1 = Curve(([0, 1], [0, 0]), 'x', 'y')

# Creation of the server
display_figure_1 = bokeh_renderer.app(figure_1, show=True, new_window=True)

# Other code blocking the execution of the cell
input("The servers will not start if you don't press any keys.")

# Creation of another figure
figure_2 = Curve(([0, 1], [0, 0]), 'x', 'y')
display_figure_2 = bokeh_renderer.app(figure_1, show=True, new_window=True)
```

**Comment by philippjfr** [comment]
Sorry you never got a reply here. This comes down to the way Jupyter works, since starting a new server adds callbacks to the Tornado event loop Jupyter is running on it cannot start until any previous tasks are completed and executing the current cell is one such task. So I'm sorry to close with a `wontfix` tag but there's really nothing we can do here.
